app/views/admin/administrators/administrators/delete.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Do you want to delete this user?</h1>
 <p class="govuk-!-font-weight-bold">Admin user: <span class="govuk-!-font-weight-regular"><%= @administrator.full_name %></span></p>
 <%= form_for :confirmation, url: admin_administrator_path(@administrator), method: :delete do |f| %>
-  <%= f.govuk_submit "Delete", classes: "govuk-button--warning" %>
+  <%= f.govuk_submit "Delete", classes: "govuk-button--warning cypress-test-delete-button" %>
```

Comment:
We discussed using `data-test="delete-button"` - but that said, I don't know if the form builder has a way of passing data attributes through

Reply:
done - used data-test

Reply:
No, that's still a class. There is currently no way of setting data attributes on formbuilder components. I'll submit a PR if I get time, this will have to do for now